<launch>
    <node name="controller" pkg="pid" type="controller" ns="roll_control" output="screen" >
      <param name="Kp" value="3.5" />
      <param name="Ki" value="0.1" />
      <param name="Kd" value="2.0" />
      <param name="upper_limit" value="1" />
      <param name="lower_limit" value="-1" />
      <param name="windup_limit" value="0.4" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
      <remap from="state" to="/roll_cur" />
      <remap from="setpoint" to="/pid_roll/control_effort" />
     </node>

     <node name="controller" pkg="pid" type="controller" ns="pitch_control" output="screen" >
      <param name="Kp" value="3.5" />
      <param name="Ki" value="0.1" />
      <param name="Kd" value="2.0" />
      <param name="upper_limit" value="1" />
      <param name="lower_limit" value="-1" />
      <param name="windup_limit" value="0.4" />
      <param name="max_loop_frequency" value="100.0" />
      <param name="min_loop_frequency" value="100.0" />
      <remap from="state" to="/pitch_cur" />
      <remap from="setpoint" to="/pid_pitch/control_effort" />
      
     </node>

</launch>
